docs/README.md

```diff
@@ -151,9 +151,9 @@ Guiding principles of state manager:
 [link to repository](https://github.com/zerobias/effector)
 
 - Effector is about atomic **stores** — it uses stateful approach that has certain problems:
-  - probable memory leaks
+  - probable memory leaks (should be proofed)
```

Comment:
```suggestion
  - probable memory leaks (as every unit of effector is a _hot_ observable, thats less safety than _cold_ observables)
```

Reply:
why less safety?

Reply:
because it has less controls for managing links (closures, etc) chain.
Cold observable read / write only with subscriptions - it kind of declarative way to manage links between observables / units.
Hot observables (like a store in effector) always connect (and process) to it producers and you may manage it only more imperative way (clearNode?) thats less safety (because not automated I mean).

Reply:
kk